### ISSUE TITLE
NPM Package?

### ISSUE THREAD

**Issue by lifenautjoe** [8bitDesigner/ng-fastclick#7]

**Comment by 8bitDesigner** [comment]
Good idea, however I've since stopped using this package now since iOS 8 removed the 800ms click delay in most cases: https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
